### PR TITLE
Fix console errors

### PR DESCRIPTION
**Pull request by pennywisexy**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
               "node_modules/bootstrap/dist/css/bootstrap.css"
             ],
             "scripts": [
+              "node_modules/jquery/dist/jquery.min.js",
```
Comment (burymm) [inline angular.json:32]: Please remove jquery

**Comment by burymm** [comment]
also plz add semicolons where it need.

File: src/app/components/film-list/film-list.component.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { Component, OnInit } from '@angular/core';
 })
 export class FilmListComponent implements OnInit {
 
-  public movies: Array<any>
+  public movies: Array<any>;
```
Comment (burymm) [inline src/app/components/film-list/film-list.component.ts:11]: Don't use any

File: angular.json (hunk 1 above)
Comment (pennywisexy) [reply angular.json:32]: after delete jquery
image

File: src/app/components/film-list/film-list.component.ts [hunk 3]
```diff
@@ -8,20 +8,40 @@ import { Component, OnInit } from '@angular/core';
...
   }
 
   public setMovie(movie) {
-    console.log()
-    this.data.currentMovie.next(movie)
+    this.data.currentMovie.next(movie);
   }
 
+  public modalDescription(event, description) {
+    const modal = document.querySelector('.modal-description');
```
Comment (burymm) [inline src/app/components/film-list/film-list.component.ts:33]: Why do you use query selector if you have angular templating? Plz use angular template to map data to descritpion.

File: src/app/components/film-list/film-list.component.ts [hunk 4]
```diff
@@ -1,27 +1,52 @@
...
-  public setMovie(movie) {
-    console.log()
-    this.data.currentMovie.next(movie)
+  public setMovie(movie): void {
+    this.data.currentMovie.next(movie);
   }
 
+  public modalDescription(item): void {
+    if(item.title !== this.title) {
+      this.title = item.title;
```
Comment (burymm) [inline src/app/components/film-list/film-list.component.ts:39]: Looks like you can simplify this
```
  this.title = item.title;
  this.showDescription = item.description;
  
   if(item.title !== this.title) {
      this.isActiveButton = true;
   } else {
      this.isActiveButton = !this.isActiveButton;
   }
```

File: src/app/components/film-list/film-list.component.ts [hunk 5]
```diff
@@ -1,27 +1,52 @@
...
+      this.showDescription = item.description;
+      this.isActiveButton = !this.isActiveButton;
+    }
+  }
 }
+
+
+
+// [ngClass]="{'active': isActiveButton}"
```
Comment (burymm) [inline src/app/components/film-list/film-list.component.ts:52]: legacy code

File: src/styles.css [hunk 6]
```diff
@@ -1,2 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
 /* @import "~bootstrap/dist/css/bootstrap.css" */
+/* body {
```
Comment (burymm) [inline src/styles.css:3]: legacy code

**Pull request merged**